### DISCUSSION TITLE
How to use in a single component, without "Vue.use(PiniaVuePlugin)" in Global

### DISCUSSION THREAD

**Discussion by z008hy** [vuejs/pinia#1173] [Help and Questions]
I use with vue2 and '@vue/composition-api'.

I'm just developing a complex component that has a lot of data that I want to manage with Pinia.

So I can't mount it on the global vue. Is there any way to solve this problem。

My development environment is.

```
    "@vue/composition-api":"^1.4.9",
    "pinia": "^2.0.12",
    "vue": "^2.6.14"
```

Big Thanks

**Comment by posva** [comment]
In short, you can't.

The user of your library must install the pinia plugin themselves.

I would say that you shouldn't use a store for this and just use ref + reactive + computed + functions.

**Reply by z008hy** [reply]
thanks
